readthedocs/search/tests/test_api.py

```diff
@@ -39,7 +42,7 @@ def test_search_works_with_title_query(self, api_client, project, page_num):
             'version': version.slug,
             'q': query
         }
-        resp = api_client.get(self.url, search_params)
+        resp = self.get_search(api_client, search_params)
```

Comment:
I don't understand this refactor -- it seems to just make it less clear what's happening?

Reply:
This is to allow us to override the api call and add HOST https://github.com/readthedocs/readthedocs.org/pull/6630/files#diff-705f0b515b1688bcb9f2a49cabc7f1a7R12